site/ee_EE.all.json

```diff
@@ -1,602 +1,602 @@
+
+  "usersubscriptions_subscription_expiry": { "other": "Aegub {{.Expiry}}" },
+  "usersubscriptions_subscription_status_cancelled": { "other": "usersubscriptions_subscription_status_cancelled" },
+  "usersubscriptions_subscription_status_errored": { "other": "usersubscriptions_subscription_status_errored" },
+  "usersubscriptions_subscription_status_past_due": { "other": "usersubscriptions_subscription_status_past_due" },
+  "usersubscriptions_subscription_status_trialing": { "other": "Prooviperiood" },
+  "usersubscriptions_subscription_unsubscribe": { "other": "Tühista minu tellimus" },
+  "usersubscriptions_unsubscribe_modal_cancel": { "other": "Tühista" },
+  "usersubscriptions_unsubscribe_modal_confirm": { "other": "Jah, tühistage minu tellimus" },
+  "usersubscriptions_unsubscribe_modal_title": { "other": "Kas tühistada tellimus?" },
+  "usersubscriptions_unsubscribe_modal_body": { "other": "See tühistab paketi {{.Name}} tellimuse. Saate vaadata paketi sisu, kuni see aegub {{.Aeguv}}." },
+
+  "shopping_card_update_reason_none": { "other": "Värskendage oma krediitkaarti. Toetatud kaardid on Visa, Mastercard ja American Express." },
+
+  "availability_coming_soon": { "other": "Tulemas" },
+  "availability_renting": { "other": "Pilet ostetud" },
+  "availability_not_available": { "other": "Pole saadaval" },
+  "availability_in_watch_window": { "other": "Saadaval" },
+  "availability_sold_out": { "other": "Välja müüdud" },
+  "availability_available_till": { "other": "Saadaval kuni {{.ExpiryDate}}" },
+  "availability_not_available_in_country": { "other": "Pole saadaval" },
+  "availability_available": { "other": "Saadaval {{.ReleaseDate}}" },
+
+  "modal_cancel": { "other": "Tühista" },
+
+  "play": { "other": "Mängi" },
+
+  "combined_auth_signin_prompt": { "other": "Logi sisse" },
+  "combined_auth_signup_prompt": { "other": "Loo konto" },
+  "combined_auth_terms": { "other": "Nõustun kõigi <a href=\"{{.TermsAndConditionsURL}}\" target=\"_blank\">kasutustingimustega</a>." },
+  "signup_terms": { "other": "Nõustun kõigi <a href=\"{{.TermsAndConditionsURL}}\" target=\"_blank\">kasutustingimustega</a>." },
+  "validation_terms_required": { "other": "Sa pead nõustuma kasutustingimustega." },
+  "shopping_error_item_limit_exceeded": { "other": "Kahjuks on {{.Title}} välja müüdud." },
+  "availability_expires": { "other": "Aegub {{.Expiry}}" },
+
+  "shopping_card_change": { "other": "Kas krediitkaart pole õige? <a href=\"{{.SubscriptionsURL}}\">Kaardi värskendamiseks klõpsake siin.</a>" },
+
+  "user_error_too_many_pin_errors": { "other": "<p>Looks like you're having trouble.</p><p>Reset your PIN or contact us for <a href=\"{{.HelpURL}}\" target=\"_blank\">help</a>" },
+
+  "app_badge_title": { "other": "Ostetud sisu vaatamiseks laadige rakendus alla!" },
+  "app_badge_ios": { "other": "Laadige alla App Store'ist" },
+  "app_badge_android": { "other": "Hankige see Google Playst" },
```

Comment:
I assume Google Play is called Google Play in all countries due to brand, prolly dont need to translate the product name here.

Reply:
Ah i see its a suffix from the language, all good